packages/Toaster/src/Toast.tsx

```diff
@@ -79,6 +83,16 @@ const Toast: React.FunctionComponent<ToastProps<ToastArgs>> = ({
                     icon={<Close size="medium" />}
                 />
             )}
+            {isClosable && getBrand() === "workleap" && (
+                <button
+                    onClick={() => {
+                        state.close(toast.key);
+                    }}
+                    className="ids-toast__close-button"
```

Comment:
I don't understand the need to change className in the context of workleap, can you tell me a bit more?

Reply:
Yep as we don't have a ghost button that has no background change on hover design suggested using a simple icon in a button instead of using the Ghost Secondary button as it would look odd.

![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/361632/c7c04008-02eb-499a-9b07-38fe4c0aaf52)

Reply:
perfect, in which case you're missing the button surrounding the icon.

Reply:
```
            {isClosable && getBrand() === "workleap" && (
                <button
                    onClick={() => {
                        state.close(toast.key);
                    }}
                    className="ids-toast__close-button"
                >
                    <Close size="medium" />
                </button>
            )}
```

What do you mean?

Reply:
Sorry my bad, 

I notice that the `{...closeButtonProps}` props aren't passed in the case of rebrand - is this intentional?

Reply:
my bad fixed